deployments/rinkeby.json

```diff
@@ -0,0 +1,17 @@
+{
+  "zAuction": [
+    {
+      "tag": "rinkeby",
+      "address": "0xe94a5456Db55af2D6a37FaDd7cEd363EDBcA65CF",
+      "version": "3ff80dd76fafda0ed3888770435964f6958e54d1754f9a2c88818000fa0897e4",
+      "date": "2022-01-20T23:43:09.943Z",
+      "args": {
+        "tokenAddress": "0x3Ae5d499cfb8FB645708CC6DA599C90e64b33A79",
```

Comment:
is this correct? I see this as zNS on kovan: https://kovan.etherscan.io/address/0x3Ae5d499cfb8FB645708CC6DA599C90e64b33A79
but under tracker here zNS doesn't show up and the code looks different: https://rinkeby.etherscan.io/address/0x3Ae5d499cfb8FB645708CC6DA599C90e64b33A79

just a sanity check, I guess this was probably generated by some script but I'm surprised this argument would be the same for rinkeby and for kovan

Reply:
![image](https://user-images.githubusercontent.com/5976499/152848599-8019ca21-f163-4e18-bf25-c3ef91219684.png)

Talked to Zach about this. There are two `CREATE` opcodes in Solidity and because the address of a deployment is a function of signer address and nonce for one of them, it is possible that this happens. We confirmed it and went to both in etherscan and they have the same nonce.